build/build.c

```diff
@@ -47,7 +47,7 @@ static rpmRC doRmSource(rpmSpec spec)
  * @todo Single use by %%doc in files.c prevents static.
  */
 rpmRC doScript(rpmSpec spec, rpmBuildFlags what, const char *name,
-		const char *sb, int test)
+	       const char *sb, StringBuf * sb_stdoutp, int test)
```

Comment:
Return value arguments should generally be the last. We failed to adjust rpmfcExec() accordingly but perhaps we could still fix this one...

Reply:
Done.